frontera/settings/__init__.py

```diff
@@ -38,7 +56,7 @@ def __setattr__(self, name, value):
             self.__dict__[name] = value
 
     def add_module(self, module):
-        if isinstance(module, Settings):
+        if isinstance(module, BaseSettings):
```

Comment:
Anyway, that's how it was before the PR. Should we include it in this PR's scope?